eppo_core/src/ufc/models.rs

```diff
@@ -18,6 +18,7 @@ pub type Timestamp = chrono::DateTime<chrono::Utc>;
 pub(crate) struct UniversalFlagConfigWire {
     /// When configuration was last updated.
     pub created_at: Timestamp,
+    pub format: AssignmentFormat,
```

Comment:
heads up @sameerank I added a new key to UFC parsing. I see it is being returned now always so I feel it is safe to make it required.